python/pyarrow/plasma.py

```diff
@@ -24,8 +24,14 @@
 import tempfile
 import time
 
-from pyarrow._plasma import (ObjectID, ObjectNotAvailable, # noqa
-                             PlasmaBuffer, PlasmaClient, connect)
+
+try:
```

Comment:
I would have rather expected this change to be in the benchmark code and not the main Python module. I think importing `pyarrow.plasma` should fail when the Plasma extension was not built. This is the clearest way to signal this problem to the user.

Reply:
This wasn't the change I intended to make (I need to fix benchmarks/plasma.py). Will fix